red-datasets.gemspec

```
@@ -0,0 +1,44 @@
+# -*- ruby -*-
+
+clean_white_space = lambda do |entry|
+  entry.gsub(/(\A\n+|\n+\z)/, '') + "\n"
+end
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), "lib"))
+require "datasets-parquet/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "red-datasets-parquet"
+  spec.version = DatasetsParquet::VERSION
+  spec.homepage = "https://github.com/red-data-tools/red-datasets-parquet"
+  spec.authors = ["otegami", "Kouhei Sutou"]
+  spec.email = ["a.s.takuya1026@gmail.com", "kou@clear-code.com"]
+
+  readme = File.read("README.md")
+  readme.force_encoding("UTF-8")
+  entries = readme.split(/^\#\#\s(.*)$/)
+  clean_white_space.call(entries[entries.index("Description") + 1])
+  description = clean_white_space.call(entries[entries.index("Description") + 1])
+  spec.summary, spec.description, = description.split(/\n\n+/, 3)
+  spec.license = "MIT"
+  spec.files = [
+    "README.md",
+    "LICENSE.txt",
+    "Rakefile",
+    "Gemfile",
+    "#{spec.name}.gemspec",
+  ]
+  spec.files += [".yardopts"]
+  spec.files += Dir.glob("lib/**/*.rb")
+  spec.files += Dir.glob("image/*.*")
+  spec.files += Dir.glob("doc/text/*")
+  spec.test_files += Dir.glob("test/**/*")
+
+  spec.add_runtime_dependency("red-parquet")
```

Could you add "red-datasets"?

```suggestion
  spec.add_runtime_dependency("red-datasets")
  spec.add_runtime_dependency("red-parquet")
```

---

Thank you so much. I couldn't imagine I should use existed gem as a dependency.🙏
fix: [df02d16](https://github.com/red-data-tools/red-datasets-parquet/pull/3/commits/df02d16b773d27b1433cfa81a8b533a28b5b9543)
fix: [9e12bf5](https://github.com/red-data-tools/red-datasets-parquet/pull/3/commits/9e12bf5e31ef4a213b71b3e9c332f29a24c41dcb)